CareKitStore/CareKitStoreTests/OCKStore/TestStore.swift

```diff
@@ -77,4 +77,46 @@ class TestStore: XCTestCase {
             XCTAssertEqual(store.context.clockTime, 2)
         }
     }
+
+    #if os(macOS)
+    func checkStoreProtection(_ protection: FileProtectionType) throws {
```

Comment:
This test is currently needed for macOS because macOS is the only one that reports these attributes directly on the file.